client/src/headingPanel.tsx

```diff
@@ -1,33 +1,101 @@
 import React, { PropsWithChildren } from "react";
 import { css } from "@emotion/react";
-import { space } from "@guardian/source-foundations";
 import { pinboard } from "../colours";
-interface HeadingPanelProps {
-  heading: string;
-  clearSelectedPinboard: () => void;
-  hasUnreadOnOtherPinboard: boolean;
-  hasErrorOnOtherPinboard: boolean;
-}
+import { palette, space } from "@guardian/source-foundations";
+import { agateSans } from "../fontNormaliser";
+import {
+  SvgChevronLeftSingle,
+  SvgCross,
+} from "@guardian/source-react-components";
+import { useGlobalStateContext } from "./globalState";
+import { PinboardData } from "./pinboard";
+import type { Tab } from "./types/Tab";
 
-export const HeadingPanel = (props: PropsWithChildren<HeadingPanelProps>) => (
-  <div
+const BackArrowIcon: React.FC = () => <SvgChevronLeftSingle size="xsmall" />;
+const CrossIcon: React.FC = () => <SvgCross size="xsmall" />;
+
+interface HeadingButtonProps {
+  onClick?: () => void;
+  icon: React.FC;
+  hoverParent?: boolean;
+}
+export const HeadingButton: React.FC<HeadingButtonProps> = ({
+  onClick,
+  icon: Icon,
+  hoverParent,
+}) => (
+  <span
+    onClick={onClick}
     css={css`
-      background-color: ${pinboard[500]};
-      padding: ${space[1]}px;
+      height: 24px;
+      width: 24px;
+      border-radius: 24px;
+      ${hoverParent ? "*:hover > &" : "&:hover"} {
+        background-color: ${pinboard[800]};
+        cursor: pointer;
+      }
+      display: flex;
+      justify-content: center;
+      align-items: center;
     `}
   >
+    <Icon />
+  </span>
+);
+
+interface HeadingPanelProps {
+  activeTab: Tab;
+  setActiveTab: (tab: Tab) => void;
+  selectedPinboard: PinboardData | undefined;
+  clearSelectedPinboard: () => void;
+}
+export const HeadingPanel = (props: PropsWithChildren<HeadingPanelProps>) => {
+  const { setIsExpanded } = useGlobalStateContext();
+  return (
     <div
       css={css`
-        font-weight: bold;
+        background-color: ${pinboard[500]};
+        border-top-left-radius: 4px;
+        border-top-right-radius: 4px;
+        box-shadow: rgba(0, 0, 0, 0.1) 0px 2px 4px;
+        clip-path: inset(0 0 -40px 0);
       `}
     >
-      <button onClick={props.clearSelectedPinboard}>
-        👈
-        {props.hasUnreadOnOtherPinboard && <sup>&nbsp;🔴</sup>}
-        {props.hasErrorOnOtherPinboard && <sup>&nbsp;⚠️</sup>}
```

Comment:
not sure we wanted to lose this functionality?!

Reply:
I've added the unread on other pinboard bubble to match the design (and almost ready to receive numbers when wired up to data). I'm not sure there's value in showing errors on other pinboards in that back button